wiki/Contests/en.md

```diff
@@ -2,291 +2,420 @@
-| 2.2 | [cYsmix - Moonlight Sonata](https://osu.ppy.sh/beatmapsets/486535) | ![osu!][osu!] | [Okoratu](https://osu.ppy.sh/users/1623405) | [Results](https://osu.ppy.sh/home/news/2018-02-15-mapping-with-rewards-2-bounties-complete) | [Score #2.2](https://docs.google.com/spreadsheets/d/e/2PACX-1vS34U2QDZTuggEIqCS4USZ2Ppq8yQk2CU_SUnI_ahJ2Dndm4S-wTD2Bhft1Say9PoRGOXDY_yCypMmO/pubhtml) |
-| 2.3 | [cYsmix - Classic Pursuit](https://osu.ppy.sh/beatmapsets/488238) | ![osu!][osu!] | [-kevincela-](https://osu.ppy.sh/users/266596) | [Results](https://osu.ppy.sh/home/news/2018-02-15-mapping-with-rewards-2-bounties-complete) | [Score #2.3](https://docs.google.com/spreadsheets/d/e/2PACX-1vS34U2QDZTuggEIqCS4USZ2Ppq8yQk2CU_SUnI_ahJ2Dndm4S-wTD2Bhft1Say9PoRGOXDY_yCypMmO/pubhtml) |
-| 3.1 | [Loki/Thaehan - A New King Is Born](https://osu.ppy.sh/beatmapsets/760694) | ![osu!][osu!] | [DTM9 Nowa](https://osu.ppy.sh/users/5428909) | [Results](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april) |  |
-| 3.2 | [Loki/Thaehan - A New King Is Born](https://osu.ppy.sh/beatmapsets/1158282) | ![osu!taiko][osu!taiko] | [Raiden](https://osu.ppy.sh/users/2239480) | [Results](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april) |  |
-| 3.3 | [Loki/Thaehan - A New King Is Born](https://osu.ppy.sh/beatmapsets/772257) | ![osu!mania][osu!mania] | [chouyaa](https://osu.ppy.sh/users/8404646) | [Results](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april) |  |
-| 3.4 | [\*namirin - closing eyes](https://osu.ppy.sh/beatmapsets/669437) | ![osu!][osu!] | [pishifat](https://osu.ppy.sh/users/3178418) | [Results](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april) |  |
-| 3.5 | [\*namirin - closing eyes](https://osu.ppy.sh/beatmapsets/705499) | ![osu!taiko][osu!taiko] | [Nardoxyribonucleic](https://osu.ppy.sh/users/876419) | [Results](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april) |  |
-| 3.6 | [Fractal Dreamers - Fortuna Redux](https://osu.ppy.sh/beatmapsets/1158282) | ![osu!][osu!] | [DTM9 Nowa](https://osu.ppy.sh/users/5428909) | [Results](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april) |  |
-
-Notes:
-
-- Mapping With Rewards #1 was concluded without a results post.
+*Main page: [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest)*
 
-## osu! Mapping Olympiad
+The Monthly Beatmaping Contest in its initial concept was a beatmapping contest where participating mappers were competing to create the most excellent mapset with a proper [difficulty spread](wiki/Beatmap/Difficulty#difficulty-levels) starting from Easy or Normal onwards of a given song (that had been cleared for use in osu! preemptively by the [osu!team](wiki/People/The_Team)) in a month's time. 
```

Comment:
```suggestion
The *Monthly Beatmaping Contest* is a beatmapping contest where participating mappers compete to create the most excellent beatmapset with a proper [difficulty spread](wiki/Beatmap/Difficulty#difficulty-levels) starting from Easy or Normal onwards of a given song in a month's time. 
```
tense should remain consistent throughout; this sounds a bit awkward

AFAICT the first paragraph should just describe what it generally *is*, and then following ones can explain differences between previous and current versions and other minutia

Reply:
given the context of this sentence i guess it should all be written in past tense, yeah ;~;